.github/workflows/test.yml

```diff
@@ -11,17 +11,17 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        backend: [compose, k8s]
+        backend: [compose]
     steps:
       - uses: actions/checkout@v3
-      - if: matrix.backend == 'compose'
```

Comment:
Did you have to remove this "if"? If we can just keep everything the same and remove "k8s" from the matrix on tests we wanna skip, that'll be cleaner and easier to undo